modules/core/src/config.ts

```diff
@@ -52,6 +52,15 @@ const formattedErc20Tokens = coins.reduce((acc: Erc20TokenConfig[], coin) => {
   return acc;
 }, []);
 
+export const ethGasConfigs = {
```

Comment:
Shouldn't it be up to the platform to enforce the min/max/default gas price/limits? How can we hard code these values in the SDK in any reasonable way, given that we can't predict future changes in the gas market?

Reply:
These values are hardcoded in platform which I copied over. However, I agree that it would be ideal to have only one source of truth and have SDK to call platform for the value. But there aren't exposed endpoints from Platform at this point and we still want to make SDK and platform's gas config consistent. 

We can certainly improve this gas configs once the routes are added to platform, but IMO this is an improvement , since we only had one hardcoded, default value in the SDK before. 

We have to get this change out for cumberland asap, what do you think about us moving forward here first and then working on the platform routes later?

Reply:
Actually, I should be able to use the getFeeEstimate route!

Reply:
SDK's fee estimation functions would use WP, so that might not be an option (unless I'm mistaken and we can source this info from a third party, I haven't verified this). If that's true, WRW would need some non-bitgo way of determining reasonable gas price/limit. I'm ok if we need to merge this with the hardcoded values you provided, but this is going to cause problems down the road if gas prices ever go real crazy again.